perl-lib/OESS/lib/OESS/Cloud.pm

```diff
@@ -113,7 +117,7 @@ sub setup_endpoints {
             my $res = $gcp->insert_interconnect_attachment(
                 interconnect_id   => $interface->cloud_interconnect_id,
                 interconnect_name => $interconnect_name,
-                bandwidth         => 'BPS_50M',
+                bandwidth         => 'BPS_' . $ep->bandwidth . 'M',
```

Comment:
I don't think we can do this for GCP.. don't they have some specific bandwidth allotments we have to pick from?